pygments/lexers/prolog.py

```diff
@@ -205,9 +205,9 @@ class LogtalkLexer(RegexLexer):
             (r'(>>|<<|/\\|\\\\|\\)', Operator),
             # Predicate aliases
             (r'\bas\b', Operator),
-            # Arithmetic evaluation
+            # Arithemtic evaluation
```

Comment:
This doesn't look right, unless I'm missing something.

Reply:
Oops! Odd typo. Fixed. Thanks for noticing.